DSA/Tree/AVLtree.cpp

```diff
@@ -0,0 +1,282 @@
+#include<stdlib.h>
```

Comment:
This is the code of Linked List , not AVL tree. Please write AVL tree Code.

Reply:
resolved